.github/workflows/ci.yml

```diff
@@ -15,23 +15,17 @@ jobs:
     strategy:
       matrix:
         react: [18, 19]
+      fail-fast: false
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast